frontend/src/Game.js

```diff
@@ -67,6 +69,7 @@ export default class Game {
                 if (count) {
                     this.spawnEnemy(enemy);
                     count--;
+                    this.enemyInfo = {...this.enemyInfo, remaining: count};
```

Comment:
Why reconstruct the entire object and not just update the remaining count instead?